### PR TITLE
Feat/card

### PR DESCRIPTION
**Pull request by k-rahul21**
add to types of cards.
also, put the value of the card.css file in the sidebar of every HTML page.

### CODE REVIEW

**Review by SJTGSHIVAM — APPROVED** [review]
Code looks clean just a minor suggestion, you can try to use % instead of vw for widh  and could also have font sizes pre-defined in some variables. Its just a suggestion so you can plan to integrate it if you fond it helpful. Everything looks good and I belive we are good to merge.

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  position: absolute;
+  top: 8px;
+  right: 5px;
+  background-color: var(--light-grey);
+  color: var(--dark-grey);
+  border-radius: 5rem;
+}
+
+.shadow-card {
+  box-shadow: 0 10px 35px rgb(0 0 0 / 10%);
```
Comment (spraveenofficial) [inline components/card/card.css]: You can move this color to Variable.

**Review by Namrata-J — APPROVED** [review]
looks good to me!! you can go ahead and merge it!!

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
+.product-name,
+.product-description,
+.price-section {
+  padding-bottom: 0.2rem;
+}
+
+.card {
+  width: 25vw;
```
Comment (veereshbv04) [inline components/card/card.css:8]: try using rem units

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+.card-cta {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.card-cta button {
+  margin: 1rem 0 0 0;
+  width: 9vw;
```
Comment (veereshbv04) [inline components/card/card.css:47]: same here use rem units instead of vw

**Pull request merged**